### PR TITLE
[Dialogs] Screenshot

### PR DESCRIPTION
**Pull request by ianegordon**

### CODE REVIEW

File: components/Dialogs/README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ path: /catalog/dialogs/
 
 # Dialogs
 
+<div class="article__asset article__asset--screenshot">
+  <img src="docs/assets/dialogs.png" alt="App Bar" width="320">
```
Comment (ajsecord) [inline components/Dialogs/README.md]: Change to 375

**Review by ajsecord — CHANGES_REQUESTED** [review]

File: components/Dialogs/README.md (hunk 1 above)
Comment (ajsecord) [reply components/Dialogs/README.md]: Also add the right alt text

**Comment by ianegordon** [comment]
PTAL

File: components/Dialogs/README.md (hunk 1 above)
Comment (ianegordon) [reply components/Dialogs/README.md]: Done

**Review by ajsecord — APPROVED** [review]

**Pull request merged**